content/rc/databases/create-database.md

```diff
@@ -53,13 +53,12 @@ between the VPC that this database is on and the VPC that the destination databa
             - Paste the certificate for your client.
     - **Data Eviction Policy** - Select a policy for evicting data when the memory limit is reached.
     - **Periodic Backups** - Enable backups and specify the [database backup]({{< relref "/rc/databases/back-up-data.md" >}}) location.
-    - **Modules** - Enable modules and select the module to use with the database.
+    - **Modules** - Enable modules and select the module to use with the database.<a name="supported-modules">&nbsp;</a>
```

Comment:
Is the supported-modules relref in the modules index intended to redirect to line 56? If so, it doesn't seem to be working for me on the staged version at [Redis Modules](https://docs.redislabs.com/staging/jira-doc-713/modules/). Instead, it seems to redirect to the "Supported modules" section at the bottom of the page.

Reply:
Working as intended.  (The anchor was placed to ensure that the "Supported modules" section was displayed.  We have a few CSS glitches and this manages past one of them. It's a temporary hack.)